src/mixins/__tests__/widget.test.js

```diff
@@ -24,20 +24,22 @@ describe('on root index', () => {
     const widget = { render: () => {} };
     const factory = jest.fn(() => widget);
     const connector = jest.fn(() => factory);
-    const Test = createFakeComponent({
-      mixins: [createWidgetMixin({ connector })],
-    });
     const widgetParams = {
       attribute: 'brand',
     };
+    const Test = createFakeComponent({
+      mixins: [createWidgetMixin({ connector })],
+      computed: {
+        widgetParams() {
+          return widgetParams;
+        },
+      },
+    });
 
     mount(Test, {
       provide: {
         $_ais_instantSearchInstance: instance,
       },
-      data: () => ({
-        widgetParams,
-      }),
     });
```

Comment:
It seems correct to have the widget mixin and widgetParams at the same level. The previous test worked with vue 2 (& VTU@1), but it didn't with vue 3. This change ensures it works with both versions.